dev-docs/bidders/kulturemedia.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: bidder
+title: KultureMedia
+description: Prebid KultureMedia Bidder Adaptor
+pbjs: true
+biddercode: kulturemedia
+media_types: banner, video
+gdpr_supported: true
+usp_supported: true
+schain_supported: true
+sidebarType: 1
```

Comment:
The more meta data you add here, the better partners can vet you.

- safeframes_ok: true/false
- deals_supported: true/false
- floors_supported: true/false
- fpd_supported: true/false
- multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
- ortb_blocking_supported: true/partial/false
- userId: (list of supported vendors)

See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter